### ISSUE TITLE
[@cap-js-community/odata-v2-adapter] Error: ENOENT: no such file or directory, lstat '/tmp/...'

### ISSUE THREAD

**Issue by vl-leon** [cap-js-community/cds-cucumber#215]
Tests break with the following error:
```
Error: ENOENT: no such file or directory, lstat '/tmp/cds_cucumber_linux/.....'
    at Object.realpathSync (node:fs:2655:7)
    at Object.<anonymous> (.../@cap-js-community/odata-v2-adapter/src/index.js:23:22)
```

**Comment by vl-leon** [comment]
@cap-js-community/odata-v2-adapter uses the temporary directory to cache model files.
To solve this issue the **TMP** environment variable should be set correctly.

**Comment by vl-leon** [comment]
released with version 0.4.1

**Comment by vl-leon** [comment]
The problem still persist when using node 20

**Comment by vl-leon** [comment]
fixed with version 0.4.2
